obelisk_ws/src/robots/achilles_model/mujoco/achilles.xml

```diff
@@ -0,0 +1,303 @@
+<!-- <?xml version="1.0" encoding="utf-8"?> -->
+<mujoco model="achilles">
+
+  <!-- Set the angles to be in radians -->
+  <compiler angle="radian" autolimits="true" eulerseq="zyx" meshdir="./meshes"/>
+
+  <!-- Fix simulation options -->
+  <option timestep="0.001" iterations="50" solver="Newton" gravity="0 0 -9.81"  integrator = "RK4"/>
+
+  <default> <geom contype="0" conaffinity="0"/> </default>.
```

Comment:
isn't this a weird default? that means the geometry default is to be visual-only

Reply:
I was following the migration steps from the changelog:
```
Removed mjOption.collision and the associated option/collision attribute.

Migration:

For models which have <option collision="all"/>, delete the attribute.

For models which have <option collision="dynamic"/>, delete all [pair](https://mujoco.readthedocs.io/en/stable/XMLreference.html#contact-pair) elements.

For models which have <option collision="predefined"/>, disable all dynamic collisions (determined via contype/conaffinity) by first deleting all [contype](https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-geom-contype) and [conaffinity](https://mujoco.readthedocs.io/en/stable/XMLreference.html#body-geom-conaffinity) attributes in the model and then setting them globally to 0 using
<default> <geom contype="0" conaffinity="0"/> </default>.
```